pkg/console/config.go

```diff
@@ -34,6 +34,7 @@ type StackConfig struct {
 	JS   webui.APIConfig `json:"js" name:"js"`
 	EDTC webui.APIConfig `json:"edtc" name:"edtc"`
 	QRG  webui.APIConfig `json:"qrg" name:"qrg"`
+	GCS  webui.APIConfig `json:"gcs" name:"gcs"`
```

Comment:
1. This is not a backwards compatible config update. When deployed (for example on Cloud Hosted) without explicitly specifying this, it will default to a localhost address, which is obviously incorrect. This means that this PR needs to go to v3.10 instead of v3.9.
2. Config changes need to be added to documentation and CHANGELOG.